playground/terraform/infrastructure/cloudbuild-manual-setup/README.md

```diff
@@ -0,0 +1,142 @@
+cloudbuild_logs_bucket_name   = "logs_bucket_name"          # Assign a globally unique name of GCP bucket for Cloud Build logs
+github_repository_name        = "beam"                      # The name of the GitHub repo. Example: In https://github.com/example/foo is 'foo'
+github_repository_owner       = "owner_name"                # Owner of the GitHub repo. Example: In https://github.com/example/foo is 'example'.
+github_repository_branch      = "branch_name"               # The name of the GitHub repo branch
+
+network_name                  = "network_name"              # GCP VPC Network Name for Playground deployment
+gke_name                      = "playground-backend"        # Playground GKE Cluster name
+state_bucket                  = "bucket_name"               # GCS bucket name for Beam Playground temp files
+bucket_examples_name          = "bucket_name-example"       # GCS bucket name for Playground examples storage
+```
+
+5. Push the `environment_name` folder to your repository.
+
+**Note:** Some regions can be prohibited for Cloud Build. Please see [Cloud Build locations](https://cloud.google.com/build/docs/locations) for supported locations.
+
+4. Run commands:
+
+```console
+# Create a new authentication configuration for GCP Project with the created user account
+gcloud init
+
+# Command imports new user account credentials into Application Default Credentials
+gcloud auth application-default login
+
+# Navigate to 01.setup folder
+cd playground/terraform/infrastructure/cloudbuild-manual-setup/01.setup/
+
+# Run terraform scripts
+terraform init -backend-config="bucket=state_bucket"
+terraform plan -var-file="../../environment/{environment_name}/terraform.tfvars"
+terraform apply -var-file="../../environment/{environment_name}/terraform.tfvars"
+```
+
+## 2. Connect GitHub repository and Cloud Build
+Follow [Connect to a GitHub repository](https://cloud.google.com/build/docs/automating-builds/github/connect-repo-github) to connect the GitHub repository and Cloud Build.
+
+## 3. Setup the Google Cloud Build triggers
+
+The `playground/terraform/infrastructure/cloudbuild-manual-setup/02.builders` provisions:
+- Cloud Build triggers to build and deploy Beam Playground.
+
+#### To execute the module
+
+```
+# Navigate to playground/terraform/infrastructure/cloudbuild-manual-setup/02.builders folder
+cd ../02.builders
+
+# Run terraform scripts
+terraform init -backend-config="bucket=state_bucket"
+terraform plan -var-file="../../environment/{environment_name}/terraform.tfvars"
+terraform apply -var-file="../../environment/{environment_name}/terraform.tfvars"
+```
+
+## 4. Running the first Cloud Build trigger to deploy Playground infrastructure
+
+1. Perform the steps described in [Prepare deployment configuration](https://github.com/apache/beam/tree/Infra%2Bplayground-in-gke/playground/terraform#prepare-deployment-configuration)
```

Comment:
Fix the link. Append "Commit and push changes to the branch {github_repository_branch} configured in terraform.tfvars

Reply:
Removed and added in Main step 1